framework/console/controllers/BaseMigrateController.php

```diff
@@ -464,10 +464,11 @@ public function actionFresh()
         if ($this->confirm(
             "Are you sure you want to reset the database and start the migration from the beginning?\nAll data will be lost irreversibly!")) {
             $this->truncateDatabase();
-            $this->actionUp();
-        } else {
-            $this->stdout('Action was cancelled by user. Nothing has been performed.');
+            return $this->actionUp();
         }
+
+        $this->stdout('Action was cancelled by user. Nothing has been performed.');
+        return ExitCode::OK;
```

Comment:
I think it should be non-ok code
For example, `echo "n" | sudo apt upgrade 1>/dev/null; echo $?` prints "1"

Reply:
That would be a backwards compatibility break.